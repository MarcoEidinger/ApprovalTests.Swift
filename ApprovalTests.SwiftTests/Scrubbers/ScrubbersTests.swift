#if os(OSX)
    @testable import ApprovalTests_Swift
#elseif os(iOS)
    @testable import ApprovalTests_iOS
#endif
import XCTest

final class ScrubbersTests: XCTestCase {

    func testSimpleScrub() throws {
        let randomNumber = Int.random(in: 1...10)
        let s = "\(randomNumber) is a nice number"
        try Approvals.verify(s, Options(ScrubWithRegEx(pattern: "\\d+", replaceWith: "<number>")))
    }
}
