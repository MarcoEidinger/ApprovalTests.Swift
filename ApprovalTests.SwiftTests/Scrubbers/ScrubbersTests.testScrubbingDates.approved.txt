{
  "breakEnd" : "<date1>",
  "breakTime" : "<date2>",
  "endOfDay" : "<date1>",
  "startTime" : "<date2>"
}
